### DISCUSSION TITLE
Save to epub to a file not visible in picker

### DISCUSSION THREAD

**Discussion by HitchhikerTTG** [plateaukao/einkbro#238] [Q&A]
Hi,

Einkbro has a great feature to export web content to an epub, and also to create / add chapters to existing epubs when they are available / visible in a picker. But when you remove a file from picker (let’s call it blogposts.epub), and then try to add new chapters to it by selecting the file from the list (list is visible when you click ‘new ePub or from picker), it overwrites the content instead of updating.

is there a way to refresh the list of files visible in picker, or is there a way to add content to existing file?

Thank you for your time and thanks in advance.

**Comment by plateaukao** [comment]
@HitchhikerTTG 
What do you mean by removing a file from picker? could you show the screen to me?
If you select "new ePub or from picker", and pick up the same file name, it should also be able to append content to it. When doing so, in the file picker, it will pop up "do you want to overwrite this file" dialog.
It'll be better that you show me with a video to describe the issue.

Also, it's not possible to refresh the list of files visible in picker. Now Android system is getting more and more picky about how Apps can access user's storage. Without user's on-purpose allowing doing so, App could not do whatever I like with these files.

>is there a way to add content to existing file?
Yes, as I said above. As long as the file is still in the storage, you can select it from file picker, and it will append content to the file.

But just keep in mind, you may need to use different reader to check if the file content is updated. Sometimes, for some reader app, it keeps file cache locally; so even file is updated, it will not reveal the difference.
